contracts/modules/claims/Incidents.sol

```diff
@@ -184,26 +200,27 @@ contract Incidents is MasterAware {
     }
 
     {
+      TokenController tc = tokenController();
       // mark cover as having a successful claim
       tc.markCoverClaimOpen(coverId);
       tc.markCoverClaimClosed(coverId, true);
 
       // create the claim
       ClaimsData cd = claimsData();
       claimId = cd.actualClaimLength();
-      cd.addClaim(claimId, coverId, coverOwner, now);
-      cd.callClaimEvent(coverId, coverOwner, claimId, now);
+      cd.addClaim(claimId, coverId, owner, now);
```

Comment:
Can we rename `owner` to `memberAddress`? `owner` is a term used for contract ownership and it's very confusing to see it here.